src/utils/mapElements.ts

```diff
@@ -30,3 +32,77 @@ export const getRevokedElements = <T extends { status: string }>(mappedData: T[]
 export const getDeletedElements = <T extends { status: string }>(mappedData: T[]) => {
   return mappedData.filter((el) => el.status === 'deleted');
 };
+
+export const mapOldAccessKeyToAccessKey = (accessKey: OldAccessKey): AccessKey => {
+  return {
+    id: accessKey.id,
+    key: accessKey.key,
+    xpubId: accessKey.xpub_id,
+    createdAt: accessKey.created_at,
+    metadata: accessKey.metadata,
+    deletedAt: accessKey.deleted_at,
+    revokedAt: accessKey.revoked_at,
+    updatedAt: accessKey.updated_at,
+  };
+};
```

Comment:
Why this mapping is necessary?

Reply:
Maybe we could make s as one task to align to the new admin api in the future.

Thanks for explanation anyway 👍😉

Reply:
I think the js-client hasn't been yet adjusted to new admin api. I was hoping after js-client will consume all new apis we can do it in the wallet-admin